llvm/lib/MC/MCObjectFileInfo.cpp

```diff
@@ -1171,8 +1172,8 @@ MCObjectFileInfo::getCallGraphSection(const MCSection &TextSec) const {
   }
 
   return Ctx->getELFSection(
-      ".callgraph", ELF::SHT_PROGBITS, Flags, 0, GroupName, true,
-      ElfSec.getUniqueID(),
+      ".llvm.callgraph", ELF::SHT_PROGBITS, Flags, 0, GroupName,
```

Comment:
I think this should be `ELF::SHT_LLVM_CALL_GRAPH` or `SHT_LLVM_CALLGRAPH` instead of `SHT_PROGBITS`.